ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/transactions/TransactionPool.java

```diff
@@ -839,41 +864,77 @@ private void executeLoadFromDisk() {
           LOG.info("Loading transaction pool content from file {}", saveFile);
           try (final BufferedReader br =
               new BufferedReader(new FileReader(saveFile, StandardCharsets.US_ASCII))) {
-            final IntSummaryStatistics stats =
+            final var stats =
```

Comment:
Would prefer to see the concrete type here rather than `var` as it's not clear what the type would be

Reply:
done